### PR TITLE
New blog post: A Test of Policies

### PR DESCRIPTION
**Pull request by lcarva**

### CODE REVIEW

File: website/content/posts/a-taste-of-policies.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+  - metadata:
+      code: github_certificate.gh_workflow_repository
+    msg: 'Repository "lcarva/festoji" not in allowed list: ["spam/spam"]'
+  - metadata:
+      code: github_certificate.gh_workflow_trigger
+    msg: 'Trigger "push" not in allowed list: ["spam"]'
+{{</code>}}
+
+As you can see, the Enterprise Contract provides a comprehensive list of all the identified
+violations.
```
Comment (simonbaird) [inline website/content/posts/a-taste-of-policies.md:137]: Idea to consider (if you didn't already). Add the `--info` flag for more detailed output, i.e. the rule titles and descriptions.

File: website/content/posts/a-taste-of-policies.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+		material.uri == "git+https://github.com/lcarva/festoji@refs/heads/master"
+	]
+	count(match) == 0
+	result := "Unexpected materials"
+}
+{{</code>}}
+
+This defines a single policy rule that verifies the materials section of the SLSA Provenance contain
+the expected git repository. The Enterprise Contract relies on rego annotations to provide
+additional information about each policy rules. See the
```
Comment (simonbaird) [inline website/content/posts/a-taste-of-policies.md]:
```suggestion
additional information about each policy rule. See the
```

or:

```suggestion
additional information about each of the policy rules. See the
```

File: website/content/posts/a-taste-of-policies.md [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+      allowed_gh_workflow_triggers:
+        - push
+  # A new policy source group uses the custom policy rules.
+  - policy:
+    - github.com/lcarva/festoji-policies//policies
+
+configuration:
+  include:
+    - github_certificate
+    - github_slsa_provenance  # Also specify which custom policy rule to include.
```
Comment (simonbaird) [inline website/content/posts/a-taste-of-policies.md:219]: Not related to this PR, but this does highlight the janky-ness of the global config.

File: website/content/posts/a-taste-of-policies.md [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+{{</code>}}
+
+We can run the ec CLI again with this new policy. The default output should be the same. Let's use
+the flag `--show-successes` to ensure the custom policy rule was included:
+
+{{<code bash>}}
+$ 🐚 ec validate image --policy policy.yaml --image $IMAGE --output yaml --show-successes
```
Comment (simonbaird) [inline website/content/posts/a-taste-of-policies.md]: Consider `--info` here too.

Comment (simonbaird) [reply website/content/posts/a-taste-of-policies.md]: You could maybe just mention it if you want to keep the output yaml brief, e.g.: 

```
To provide more details about the rules, including their titles and descriptions, you can use the `--info` flag.
```

**Review by simonbaird — APPROVED** [review]
Fantastic! :heart: :heart:

File: website/content/posts/a-taste-of-policies.md (hunk 3 above)
Comment (lcarva) [reply website/content/posts/a-taste-of-policies.md:219]: Yeah, that's how I came across it :sweat_smile:

File: website/content/posts/a-taste-of-policies.md (hunk 1 above)
Comment (lcarva) [reply website/content/posts/a-taste-of-policies.md:137]: Yeah, that's a good idea. Thanks!

File: website/content/posts/a-taste-of-policies.md (hunk 2 above)
Comment (lcarva) [reply website/content/posts/a-taste-of-policies.md]: Opted for the second one.

File: website/content/posts/a-taste-of-policies.md (hunk 4 above)
Comment (lcarva) [reply website/content/posts/a-taste-of-policies.md]: Let's go with the `--info` output. I shrank down the list of successes. There's no need to show all the github ones again.

**Pull request merged**